website/docs/r/organization.html.markdown

```diff
@@ -33,13 +33,7 @@ resource "mongodbatlas_organization" "test" {
 ~> **NOTE:** Creating an organization will return a new API Key pair that can be used to authenticate and manage the new organization  with MongoDB Atlas Terraform modules/blueprints.  You cannot use the newly created API key pair to manage the newly created organization in the same Terraform module/blueprint that the organization is created in.
 
 
-* `role_names` - (Required) List of Organization roles that the Programmatic API key needs to have. Ensure you provide: at least one role and ensure all roles are valid for the Organization.  You must specify an array even if you are only associating a single role with the Programmatic API key.
- The following are valid roles:
-  * `ORG_OWNER`
-  * `ORG_GROUP_CREATOR`
-  * `ORG_BILLING_ADMIN`
-  * `ORG_READ_ONLY`
-  * `ORG_MEMBER` 
+* `role_names` - (Required) List of Organization roles that the Programmatic API key needs to have. Ensure you provide: at least one role and ensure all roles are valid for the Organization.  You must specify an array even if you are only associating a single role with the Programmatic API key. The [MongoDB Documentation](https://www.mongodb.com/docs/atlas/reference/user-roles/#organization-roles) describes the roles a user can have.
```

Comment:
minor edits for clarity. Because this is in the context of an API key I think we can update the docs reference to mention that.
```suggestion
* `role_names` - (Required) List of Organization roles that the Programmatic API key needs to have. Ensure that you provide at least one role and ensure all roles are valid for the Organization.  You must specify an array even if you are only associating a single role with the Programmatic API key. The [MongoDB Documentation](https://www.mongodb.com/docs/atlas/reference/user-roles/#organization-roles) describes the roles that you can assign to a Programmatic API key.
```